rust/scx_utils/src/topology.rs

```diff
@@ -95,6 +95,13 @@ lazy_static::lazy_static! {
     pub static ref NR_CPUS_POSSIBLE: usize = libbpf_rs::num_possible_cpus().unwrap();
 }
 
+#[derive(Debug, Clone, Eq, Hash, PartialEq)]
+pub enum CoreType {
+    PCore,
+    ECore,
+    Unknown,
```

Comment:
Not a huge fan of the naming, so if someone can think of a better naming convention.

Reply:
Would `Big` and `Little` be better? Those are the OG terms and a bit wider spread, I think.

Reply:
Yeah, calling `is_ecore` just seemed wrong 😄

Reply:
Another nitpick. Why not `CoreType::Big` and `CoreType::Little`? Do we ever import specific enum values?

Reply:
It would be nice to have a `Turbo` type. When there are two or more types of CPU with different max freq, the max freq CPUs are turbo bootable CPUs. This is true at least Intel Raptor Lake and Qualcomm Snapdragon platforms.